triage/render.js

```diff
@@ -197,42 +244,82 @@ function renderCluster(top, key, keyId, text, tests) {
     createElement('pre', null, options.showNormalize ? key : text),
     createElement('div', {className: 'graph', dataset: {cluster: keyId}}),
   ]);
+  var latest = createElement('table');
   var list = addElement(failureNode, 'ul', null, [
-    createElement('li', {innerText: `Latest Failures ${rightArrow}`}, [
-      createElement('div', {className: 'latest', dataset: {cluster: keyId}})
-    ]),
+    createElement('span', null, [`Latest Failures`, latest]),
   ]);
 
+  renderLatest(latest, keyId);
+
   var clusterJobs = addElement(list, 'li');
 
   var jobSet = new Set();
 
   var testList = createElement('ul');
-  if (tests.length > kCollapseThreshold) {
-    testList.style.display = 'none';
-  }
 
-  addElement(list, 'li', null, [`Failed in ${plural(tests.length, 'Test', 's')} ${pickArrow(tests.length)}`, testList]);
+  var expander = addElement(list, 'li', null, [`Failed in ${plural(tests.length, 'Test', 's')} ${downArrow}`, testList]);
 
   // If we expanded all the tests and jobs, how many rows would it take?
   var jobCount = sum(tests, t => t.jobs.length);
 
+  // Sort tests by descending [last day hits, total hits]
+  var tests = sortByKey(tests, t => [-dayCounts(counts[t.name]), -sum(t.jobs, j => j.builds.length)]);
+
+  var allTestsDayCount = dayCounts(counts['']);
+  var testsDayCountSum = 0;
+
+  var allSection = false;
+  var testsShown = 0;
+  var i = 0;
+
   for (var test of tests) {
+    i++;
     var testCount = sum(test.jobs, j => j.builds.length);
+
+    var testDayCount = dayCounts(counts[test.name]);
+
+    if (tests.length > kCollapseThreshold) {
+      if (!allSection && testsDayCountSum > 0.8 * allTestsDayCount) {
+        testsShown = i;
+        addElement(testList, 'button', {className: 'rest'}, 'Show Daily Bottom 20% Tests');
```

Comment:
Something simplier please. Maybe `More`?

If you need, put the details in the `title="Shows bottom 50% of results"` so if someone neurotic hovers over the button it will give them details.